src/test/projects/basic/pom.xml

```diff
@@ -34,7 +34,7 @@
             <plugin>
                 <groupId>com.github.carlomorelli</groupId>
                 <artifactId>licensescan-maven-plugin</artifactId>
-                <version>1.1</version>
+                <version>2.0</version>
```

Comment:
So apparently we can use the ${it-project.version} variable here and the test harness injects the current version. Ok to do later if you want to split it out

Reply:
I tried but doesn't work on my locale. Don't know why. let's keep it for future. opening #12